src/Sarif.TestUtilities/FileDiffingTests.cs

```diff
@@ -75,7 +75,7 @@ protected virtual void RunTest(string inputResourceName, string expectedOutputRe
             bool passed;
             if (_testProducesSarifCurrentVersion)
             {
-                PrereleaseCompatibilityTransformer.UpdateToCurrentVersion(expectedSarifText, forceUpdate: false, Formatting.Indented, out expectedSarifText);
+                PrereleaseCompatibilityTransformer.UpdateToCurrentVersion(expectedSarifText, forceUpdate: true, Formatting.Indented, out expectedSarifText);
```

Comment:
>true [](start = 106, length = 4)

Some tests were failing with deserialization errors because without this, the core transformations didn't happen.

Reply:
ok. suggest resetting this value to false. there are three inputs that are failing. go update their version to 10-10, which will provoke the transform. i think this is better than having test files that profess to be 11-28 but which clearly aren't

---
In reply to: [248880334](https://github.com/Microsoft/sarif-sdk/pull/1221#discussion_r248880334) [](ancestors = 248880334,248876087)

Reply:
Good idea. Two of the files actually were already 10-10 and needed to be downgraded to 09-26 to provoke the core transform, but all tests now pass with `forceUpdate: false`.

---
In reply to: [248887247](https://github.com/Microsoft/sarif-sdk/pull/1221#discussion_r248887247) [](ancestors = 248887247,248880334,248876087)